docs/operating-system/configuration.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Home Assistant Operating System configuration"
+sidebar_label: OS Configuration
```

Comment:
Since this is already under Operating System, "OS" is duplication here:

```suggestion
sidebar_label: Configuration
```

Reply:
OS doesn't really help a lot to clarify what type of configuration we are talking here, e.g. my initial thought was that this describes the build time operating system configuration (as in Buildroot defconfig) since it is the developer documentation :shrug: .

But that is a general problem that the remaining docs we had in the OS repo was mostly user focused.